flow/connectors/postgres/cdc.go

```diff
@@ -35,7 +35,7 @@ type PostgresCDCSource struct {
 	customTypeMapping      map[uint32]string
 
 	// for partitioned tables, maps child relid to parent relid
-	chIdToParRelId map[uint32]uint32
+	childToParentRelIDMapping map[uint32]uint32
```

Comment:
Splitting renaming to a separate PR makes it easier to review PR & `git blame` more effective